### DISCUSSION TITLE
TabControl not showing tab headers

### DISCUSSION THREAD

**Discussion by drei97** [AvaloniaUI/Avalonia#11218] [Q&A] [answered]
Hello,

I am still with **0.10.19**!

The follwing XAML is given:

`         
          <TabControl Background="Aqua">

          <TabControl Background="Aqua">
                <TabItem Background="Bisque">
                    <TabItem.Header>
                            <TextBlock Text="one"/>
                    </TabItem.Header>
                    <Label Content="1"/>
                </TabItem>
                <TabItem Header="____two____" Background="Brown">
                    <Label Content="2"/>
                </TabItem>              
            </TabControl>
`

which gives the following 

![grafik](https://user-images.githubusercontent.com/68755045/235903209-12f4b14d-a384-406f-b78b-d7169f8aa71e.png)

Why are the tab headers not filled with text?

**Comment by timunie** [comment]
Works for me: 

![image](https://user-images.githubusercontent.com/47110241/235906900-6800973e-257b-406e-9ed1-848b93df5bcb.png)

tested using 0.10.19 and latest 11.0 preview

**Reply by drei97** [reply, marked as answer]
OK, it seems that the problem is outside of that snipped of code. Makes me much more confused!

....

I found a typo in my styles! grrrr


Everything is fine now.
